controllers/auth.js

```diff
@@ -54,10 +54,28 @@ const refresh = async (req, res) => {
   res.status(200).json(tokens)
 }
 
+const sendResetPasswordEmail = async (req, res) => {
+  const { email } = req.body;
+
+  const resetToken = await authService.sendResetPasswordEmail(email)
+
+  res.status(200).json({ resetToken });
```

Comment:
I don't think we need to return resetToken for security reasons

Reply:
Well, considering security reasons, I send this resetToken as part of a reset link, isn’t that the same as sending it back to the client?
And if not send the reset token, what to send?

Reply:
Ok, I got you.
But if we can avoid exposing that data in one place, we should do that, and we don't need resetToken on client.
So, we can just send 204 status code

Reply:
Got it, will update soon